zlist_expand should not return a value

```c
// src/zlist.c:13:13: error: void function 'zlist_expand' should not return a value
     return 0;
//   ^      ~
```

Just replacing `return 0;` with `return;` fixes it.

--- a/src/zlist.c
+++ b/src/zlist.c
@@ -10,7 +10,7 @@ static void zlist_expand(void **list, size_t elemsize) {
         void *tmp = zrealloct(*list, 1 + zalloc(*list) * 2, elemsize);
         if(!tmp) {
             throw("Error while reallocating list!");
-            return 0;
+            return;
         }
         *list = tmp;
     }

